src/components/MoneyReportHeader.tsx

```diff
@@ -162,13 +180,90 @@ function MoneyReportHeader({policy, report: moneyRequestReport, transactionThrea
         TransactionActions.markAsCash(iouTransactionID, reportID);
     }, [requestParentReportAction, transactionThreadReport?.reportID]);
 
+    const changeMoneyRequestStatus = () => {
+        if (!transactionThreadReport) {
+            return;
+        }
+        const iouTransactionID = requestParentReportAction?.actionName === CONST.REPORT.ACTIONS.TYPE.IOU ? requestParentReportAction.originalMessage?.IOUTransactionID ?? '' : '';
+
+        if (isOnHold) {
+            IOU.unholdRequest(iouTransactionID, transactionThreadReport.reportID);
+        } else {
+            const activeRoute = encodeURIComponent(Navigation.getActiveRouteWithoutParams());
+            Navigation.navigate(ROUTES.MONEY_REQUEST_HOLD_REASON.getRoute(policy?.type ?? CONST.POLICY.TYPE.PERSONAL, iouTransactionID, transactionThreadReport.reportID, activeRoute));
+        }
+    };
+
+    const getStatusIcon: (src: IconAsset) => React.ReactNode = (src) => (
+        <Icon
+            src={src}
+            height={variables.iconSizeSmall}
+            width={variables.iconSizeSmall}
+            fill={theme.icon}
+        />
+    );
+
+    const getStatusBarProps: () => MoneyRequestHeaderStatusBarProps | undefined = () => {
+        if (hasOnlyHeldExpenses) {
+            return {title: translate('iou.hold'), description: translate('iou.expensesOnHold'), danger: true, shouldShowBorderBottom: true};
+        }
+        if (allHavePendingRTERViolation) {
+            return {title: getStatusIcon(Expensicons.Hourglass), description: translate('iou.pendingMatchWithCreditCardDescription'), shouldShowBorderBottom: true};
+        }
+    };
+
+    const statusBarProps = getStatusBarProps();
+
     // The submit button should be success green colour only if the user is submitter and the policy does not have Scheduled Submit turned on
     const isWaitingForSubmissionFromCurrentUser = useMemo(
         () => chatReport?.isOwnPolicyExpenseChat && !policy?.harvesting?.enabled,
         [chatReport?.isOwnPolicyExpenseChat, policy?.harvesting?.enabled],
     );
 
     const threeDotsMenuItems = [HeaderUtils.getPinMenuItem(moneyRequestReport)];
+    if (canHoldOrUnholdRequest) {
+        const isRequestIOU = chatReport?.type === 'iou';
```

Comment:
Or we can just reuse `ReportUtils.isIOUReport`
```suggestion
        const isRequestIOU = chatReport?.type === CONST.REPORT.TYPE.IOU;
```